src/main/java/org/mitre/synthea/export/BFDExportConfigEntry.java

```diff
@@ -0,0 +1,94 @@
+package org.mitre.synthea.export;
+
+import java.math.BigDecimal;
+import java.io.Reader;
+
+/** A single line in the export configuration TSV file, containing the full properties for a specification 
+ *  Implementation Note: the names of the properties MUST be exactly the same as the heading in the TSV file
+*/
+public class BFDExportConfigEntry {
+  private String field;
+  private String beneficiary;
+  private String beneficiary_history;  
+  private String inpatient;
+  private String outpatient;
+  private String carrier;
+  private String prescription;
+
+  /** trims any newlines, tabs, spaces from value 
+   *  @param value the string to evaluate for newlines, tabs, etc.
+  */
+  public String trimAllWhitespace( String value ) {
```

Comment:
Should probably be private.

Reply:
And since it is a single line of code -- not sure why you even need the function whatsoever.

Reply:
I may need to extend this.  It used to use something that is not available in Java 8.  I like to keep this until the next PR.